GPy/kern/src/stationary.py

```diff
@@ -426,6 +425,23 @@ class OU(Stationary):
     def __init__(self, input_dim, variance=1., lengthscale=None, ARD=False, active_dims=None, name='OU'):
         super(OU, self).__init__(input_dim, variance, lengthscale, ARD, active_dims, name)
 
+    def to_dict(self):
+        """
+        Convert the object into a json serializable dictionary.
+
+        Note: It uses the private method _save_to_input_dict of the parent.
```

Comment:
Not really necessary note, the method's code is crystal clear so anyone interested in this level of detail won't need these additional lines.